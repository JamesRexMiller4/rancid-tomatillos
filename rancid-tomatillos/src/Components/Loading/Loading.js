import React from 'react';
import './Loading.scss';

const Loading = () => {
  return ( 
    <div className='loading-wrapper-div'>
      <div className='popcorn-div'></div>
    </div>
   );
}
 
export default Loading;