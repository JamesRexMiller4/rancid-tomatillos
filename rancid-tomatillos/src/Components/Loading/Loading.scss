.loading-wrapper-div {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100vw;
  height: 100vh;
  background: {
    image: url('../../images/Red\ Curtain1.jpg');
    repeat: no-repeat;
    size: cover;
  }
}

.popcorn-div {
  width: 25vw;
  height: 25vw;
  background: {
    image: url('../../images/popcorn.png');
    size: cover;
    repeat: no-repeat;
  }
  animation: pulse 2s infinite;
}

@keyframes pulse {
	0% {
		transform: scale(0.95);
	}
	
	70% {
		transform: scale(1);
	}
	
	100% {
		transform: scale(0.95);
	}
}