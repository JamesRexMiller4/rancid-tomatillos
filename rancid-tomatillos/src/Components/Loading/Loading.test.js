import React from 'react';
import {shallow} from 'enzyme';
import Loading from './Loading.js'
 
describe('Loading', () => {
  let wrapper;
 
  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })
 
  it('should match the snapshot with all the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  });
 
});