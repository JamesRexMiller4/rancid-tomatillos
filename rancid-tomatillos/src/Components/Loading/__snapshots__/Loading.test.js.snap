// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
<div
  className="loading-wrapper-div"
>
  <div
    className="popcorn-div"
  />
</div>
`;
